frontend/src/pages/org/archived-item-detail/archived-item-detail.ts

```diff
@@ -148,8 +154,8 @@ export class ArchivedItemDetail extends TailwindElement {
     return activeCrawlStates.includes(this.crawl.state);
   }
 
-  private get isQAActive(): boolean | null {
-    if (!this.qaRuns?.[0]) return null;
+  private getIsQAActive(): boolean {
```

Comment:
I don't think this needs to be a function, it's only used once and could be inlined as `this.qaRuns?.[0] && QA_RUNNING_STATES.includes(this.qaRuns[0].state)`

Reply:
this was to make isQAActive a property to cause a rerender when it changes, but maybe there's a simpler way?

Reply:
Yeah, it makes sense to have it be a a state property and update it within `fetchQARuns`, it's just the calculation of that value that could be inlined I think

Reply:
Updated!